compiler/rustc_interface/src/util.rs

```diff
@@ -670,6 +670,24 @@ pub fn build_output_filenames(
     }
 }
 
+#[cfg(not(target_os = "linux"))]
+pub fn non_durable_rename(src: &Path, dst: &Path) -> std::io::Result<()> {
+    std::fs::rename(src, dst)
+}
+
+/// This function attempts to bypass the auto_da_alloc heuristic implemented
```

Comment:
nit: the wrapping of lines in this documentation comment seems quite inconsistent, if you'd like to change that then I wouldn't mind :)